ide-eclipse.html

```diff
@@ -0,0 +1,347 @@
+</p>
+<p>
+    Include the new JDK as <kbd>Installed JREs</kbd> in <kbd>Eclipse -> Preferences -> Java -> Installed JREs -> Add</kbd>.
+</p>
+
+<div class="alert alert-warning">
+    <strong>Note: </strong>
+    If you get this message when you add JDK 11:
+
+<pre><code>
+You selected a JRE that this version of Eclipse JDT does not yet support fully. Some of the features may not work as expected.
+</code></pre>
+    
+then install the patch from the Eclipse MarketPlace: 
+<a href="https://marketplace.eclipse.org/content/java-11-support-eclipse-2018-09-49">Java 11 support for Eclipse 2018-09 (4.9)</a>.
+</div>
+
+<p>
+    You can create a JavaFX 11 project and use <a href="#IDE">IDE</a>,
+    <a href="#Maven">Maven</a> or <a href="#Gradle">Gradle</a> build tools.
+</p>
+
+<a name="IDE"></a><h3>IDE</h3>
+
+<p>
+    Follow these steps to create a JavaFX project and use the IDE tools to build it and run it.
+</p>
+
+<p>
+    Download the appropriate
+    <a target="_blank" href="https://gluonhq.com/products/javafx/">JavaFX SDK</a>
+    for your operating system and unzip it to a desired location, for instance
+    <kbd>/Users/your-user/Downloads/javafx-sdk-11</kbd>.
+</p>
+
+<p>
+    Create a new <kbd>User Library</kbd> under <kbd>Eclipse -> Window -> Preferences -> Java -> Build Path -> User Libraries -> New</kbd>. 
+    
+    <a href="images/ide/eclipse/ide/eclipse01.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse01.png" alt="Add User Library"/></a>
+   
+    Name it <kbd>JavaFX11</kbd> and include the jars under the <kbd>lib</kbd> folder from JavaFX 11.
+    
+    <a href="images/ide/eclipse/ide/eclipse02.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse02.png" alt="Add JavaFX jars"/></a>
+   
+</p>
+
+<p>
+    <strong>1. Create a Java project</strong>
+    
+    <a href="images/ide/eclipse/ide/eclipse03.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse03.png" alt="Create a Java project"/></a>
+   
+    Provide a name to the project, like <kbd>HelloFX</kbd>, and a location. Make sure JDK 11 is selected.
+    You don't need to add a <kbd>module-info.java</kbd> file. You can include the <kbd>JavaFX11</kbd> library.
+    
+    <a href="images/ide/eclipse/ide/eclipse04.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse04.png" alt="Add JavaFX11 library"/></a>
+
+    An empty project will be opened.
+</p>
+
+<p>
+    <strong>2. Add JavaFX classes</strong>
+</p>
+<p>    
+    You can add a package named <kbd>hellofx</kbd>, a main class <kbd>Main</kbd>, with and FXML file and a controller.
+    
+    <a href="images/ide/eclipse/ide/eclipse05.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse05.png" alt="Add JavaFX project"/></a>
+</p>
+
+<div class="alert alert-warning">
+    <strong>Warning: </strong>
+    If you run now the project it will compile but you will get this error:
+
+<pre><code>
+Error: JavaFX runtime components are missing, and are required to run this application
+</code></pre>
+
```

Comment:
Maybe we need a short explanation on this?
e.g. "This error is shown since the Java 11 launcher checks if the main class extends javafx.application.Application. If it does so, it is required to have the javafx.graphics module on the module-path"